plugins/modules/azure_rm_access_token.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2023 Patrick Uiterwijk <@puiterwijk>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_access_token
+
+version_added: "1.14.0"
+
+short_description: Get Azure API access token
+
+description:
+    - Get an access token for Azure APIs.
+
+options:
+    scopes:
+        description:
+            - The scopes to request.
+        type: list
+        required: True
+    claims:
+        description:
+            - Additional claims required in the token.
+        type: list
+    token_tenant_id:
+        description:
+            - Tenant to include in the token request.
+        type: str
+    enable_cae:
+        description:
+            - Whether to enable Continuous Access Evaluation (CAE) for the requested token.
+        default: False
+        type: bool
+
+extends_documentation_fragment:
+    - azure.azcollection.azure
+
+author:
+    - Patrick Uiterwijk (@puiterwijk)
+'''
+
+EXAMPLES = '''
+- name: Get access token for Microsoft Graph
+  azure.azcollection.azure_rm_access_token:
+    scopes:
+    - https://graph.microsoft.com/.default
```

Comment:
Wrong indentation: expected 6 but found 4
```suggestion
      - https://graph.microsoft.com/.default
```